tests/integration/test_vulnerability_detector/test_feeds/canonical/test_extra_tags_canonical_feed.py

```diff
@@ -0,0 +1,96 @@
+
+# Copyright (C) 2015-2020, Wazuh Inc.
+# Created by Wazuh, Inc. <info@wazuh.com>.
+# This program is free software; you can redistribute it and/or modify it under the terms of GPLv2
+
+import os
+import pytest
+
+from wazuh_testing.tools import LOG_FILE_PATH
+from wazuh_testing.tools.monitoring import FileMonitor
+from wazuh_testing.tools.configuration import load_wazuh_configurations
+from wazuh_testing.tools import file
+from wazuh_testing.tools.utils import insert_xml_tag
+from wazuh_testing.tools.services import control_service
+import wazuh_testing.vulnerability_detector as vd
+
+# Marks
+pytestmark = pytest.mark.tier(level=2)
+
+current_test_path = os.path.dirname(os.path.realpath(__file__))
+test_data_path = os.path.join(current_test_path, '..', '..', 'data')
+configurations_path = os.path.join(test_data_path, 'configuration', 'test_feeds', vd.INVALID_CANONICAL_FEEDS_CONF)
+custom_canonical_oval_feed_path = os.path.join(test_data_path, 'feeds', vd.CUSTOM_CANONICAL_OVAL_FEED)
+
+wazuh_log_monitor = FileMonitor(LOG_FILE_PATH)
+
+# Set configuration
+parameters = [{'CANONICAL_CUSTOM_FEED': custom_canonical_oval_feed_path}]
+ids = ['CANONICAL_configuration']
+
+test_data = [[1, 2, 3], {"a": 1, "b": 2}, "extra_tag", 12345, "ñ", "テスト", "ИСПЫТАНИЕ", "测试", "اختبار", " ", ""]
+test_values = [(tag, value) for tag in test_data for value in test_data]
```

Comment:
These lines are used in other tests. Maybe it would be interesting to move them to the vulnerability detector module and import them.

Reply:
Done in ba7bcc0729d9649d5d646f0d5ea80a440267f65b